packages/aws-cdk-lib/aws-ec2/lib/instance-types.ts

```diff
@@ -1632,6 +1642,8 @@ export class InstanceType {
       [InstanceClass.MAC2_M2]: 'mac2-m2',
       [InstanceClass.MACINTOSH2_M2_PRO]: 'mac2-m2pro',
       [InstanceClass.MAC2_M2PRO]: 'mac2-m2pro',
+      [InstanceClass.MACINTOSH2_M1_ULTRA]: 'macintosh2-m1-ultra',
```

Comment:
Where does this string come from? I cannot find [instance type](https://aws.amazon.com/ec2/instance-types/) named macintosh2-m1-ultra.